src/toolchain/toolchain.ts

```diff
@@ -198,17 +198,22 @@ export class SwiftToolchain {
                 return path.join(stdout.trimEnd(), "usr", "bin");
             }
             case "win32": {
+                // look up runtime library directory for XCTest alternatively
+                const fallbackPath =
```

Comment:
Can we remove this change it is unrelated.

Reply:
This is just moved here from L207-209